scripts/push_pkg.sh

```diff
@@ -0,0 +1,129 @@
+#!/bin/sh
+
+set -e
+
+topdir="$(realpath "$(dirname "${0}")/../openwrt")"
+
+# defaults to qemu run script
+ssh_port=2223
+ssh_host=localhost
+build_only=0
+
+print_help() {
+	echo "$0 [OPTIONS] PACAKGE_DIR"
+	echo ""
+	echo " -h          print this help"
+	echo " -r HOST     use ssh host (default port will be 22)"
+	echo " -p PORT     use ssh port"
+	echo " -b          build only, do not push anywhere"
+	echo ""
+	echo ' To change gluon variables, run e.g. "make config GLUON_MINIFY=0"'
+	echo ' because then the gluon logic will be triggered, and openwrt/.config'
+	echo ' will be regenerated. The variables from openwrt/.config are already'
+	echo ' automatically used for this script.'
+	echo
+}
+
+while getopts "p:r:hb" opt
+do
+	case $opt in
+		p) ssh_port="${OPTARG}";;
+		r) ssh_host="${OPTARG}"; ssh_port=22;;
```

Comment:
The behaviour of the option handling is somewhat confusing... If `-r` is used after `-p`, the `-p` will be ignored.

How about selecting the port after all options have been parsed, using the following logic?

- If `-p` was given, always use that port
- Else-if `-r` was given, default to port 22
- Else, default to port 2223